conans/client/tools/oss.py

```diff
@@ -24,14 +24,35 @@ def args_to_string(args):
         return " ".join("'" + arg.replace("'", r"'\''") + "'" for arg in args)
 
 
+class CpuProperties(object):
```

Comment:
- Do not use bare exceptions, they capture Ctrl+C and other interrupts too

This is a bit too much Object Oriented for me without a real goal:
- All methods should be static (and are not, this is an indicator)
- It is mainly designed for testing with mocks, which I get, but you can get the same with one function:

```python
 def get_cpus(quota_file=None, period_file=None):
        try:
            cfs_quota_us = int(load(quota_file or "/sys/fs/cgroup/cpu/cpu.cfs_quota_us"))
            cfs_period_us = int(load(period_file or "/sys/fs/cgroup/cpu/cpu.cfs_period_us"))
            if cfs_quota_us > 0 and cfs_period_us > 0:
                return int(math.ceil(cfs_quota_us / cfs_period_us))
        except Exception:
            pass
        return multiprocessing.cpu_count()
```

Now it is easy to test too, and also testing is more real, it test also the load of file + conversion to integer